frontend/src/routes/Governance/common/util.tsx

```diff
@@ -703,39 +704,46 @@ export function getPolicySource(
   return source
 }
 
-export function getEngineString(kind: string): string {
-  switch (kind) {
-    case 'ConfigurationPolicy':
-    case 'CertificatePolicy':
-    case 'OperatorPolicy':
+export function getEngineString(apiGroup: string): string {
+  switch (apiGroup) {
+    case 'policy.open-cluster-management.io':
       return 'Open Cluster Management'
+    case 'constraints.gatekeeper.sh':
```

Comment:
Can you add constraintTempate too? because of this https://github.com/stolostron/policy-collection/blob/6f7996f4c2abdc6da70b950b0176517bb1ad48d2/community/AC-Access-Control/policy-gatekeeper-limitclusteradmin.yaml#L37